pages/1.11/administering-clusters/convert-agent-type/index.md

```diff
@@ -62,7 +57,7 @@ You can determine the node type by running this command from the DC/OS CLI.
     ```
 
 ### Install DC/OS and convert agent node
-Copy the archived DC/OS installer file (`dcos-install.tar`) to the node that that is being converted. This archive is created during the GUI or CLI [installation](/1.11/installing/oss/custom/gui/#backup) method.
+Copy the archived DC/OS installer file (`dcos-install.tar`) to the node that that is being converted. This archive is created during the GUI or CLI [installation](/1.11/installing/evaluation/cloud-installation/) method.
```

Comment:
Replacing /1.11/installing/oss/custom/gui/#backup with /1.11/installing/evaluation/cloud-installation/ is incorrect.  /1.11/installing/evaluation/cloud-installation/#backup is the correct link.

Reply:
Since there is no #backup link on /1.11/installing/evaluation/cloud-installation/, I have deleted it. It really does not make any difference, however, whether the #backup tag is in the URL, since the link will take you to the same page with or without it.